lte/gateway/c/session_manager/SetMessageManagerHandler.h

```diff
@@ -83,6 +83,12 @@ class SetMessageManagerHandler : public SetMessageManager {
       const SetSmNotificationContext& notif,
       std::function<void(Status, SmContextVoid)> response_callback);
 
+  /* When service request is received from AMF after paging request sent
+   */
```

Comment:
`Handle service request received from AMF after paging request is sent` maybe? Let's try to use imperative mode in docs.

Reply:
Updated the comment as suggested.